data/step_function_inputs/S1_RTC_seasonal_composite.json

```diff
@@ -0,0 +1,7 @@
+{
+    "collection": "s1-rtc-seasonal-composite",
+    "discovery": "inventory",
+    "inventory_url": "s3://maap-ops-workspace/shared/nathanmthomas/sentinel1_seasonal_comps/inventory.csv",
+    "cogify": false,
```

Comment:
[Not a breaking change] Defaults for cogify and upload are `false` - only mentioning this because the icesat2-boreal stepfn config doesn't include the cogify default. Not sure if you want them uniform in the way that they show all values even if default or if they don't show values unless different than the default